app/Http/Requests/Api/Admin/OrganizationAssociateUsersRequest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Http\Requests\Api\Admin;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class OrganizationAssociateUsersRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
+    {
+        return true;
```

Comment:
Qualquer usuário pode associar usuários a organizações?

Reply:
Ok, acho que entendi, a restrição está na rota via middleware "only_admin_user". Se for isso mesmo, ignore o comentário.